doc/lua_api.txt

```diff
@@ -2226,6 +2226,17 @@ Elements
 * `frame duration`: Milliseconds between each frame. `0` means the frames don't advance.
 * `frame start` (Optional): The index of the frame to start on. Default `1`.
 
+### `model[<X>,<Y>;<W>,<H>;<mesh name>;<texture names>;<rotation X,Y>;<continuous>;<mouse control>]`
+
+* Show a mesh model.
+* `mesh name`: The mesh model to use.
+* `texture names`: The mesh textures to use according to the mesh materials.
+   Texture names must be separated by commas.
+* `rotation {X,Y}` (Optional): Initial rotation of the camera.
+  The axes are euler angles in degrees.
```

Comment:
They are in radians, I tested it. Or did you change it?

Reply:
Y was taken as rotation speed, which is a bit strange because controllable speed was not intended in the original PR.
It's degrees for sure.